sematic/api_client.py

```diff
@@ -89,6 +88,20 @@ def _get_artifact(artifact_id: str) -> Artifact:
     return Artifact.from_json_encodable(response["content"])
 
 
+def store_artifact_bytes(artifact_id: str, bytes_: bytes) -> None:
+    response = _get(f"/artifacts/{artifact_id}/location")
+
+    location: str = response["location"]
+
+    put = requests.put if location.startswith("https://") else _put
```

Comment:
The issue is that `_put` presumes that the location passed is a Sematic API endpoint. Therefore, it will prepend `/api/v1` and also add authentication HTTP headers.

The contract here is that the location returned by the server is either a fully-qualified URL (`https://...`), in which case we simply query it with `requests.pu`, or it is a relative API path, in which case we pass to the usual `_put`.